ploneorg/core/content/plonerelease.py

```diff
@@ -74,9 +79,56 @@ class IPloneRelease(Schema):
         required=False
     )
 
+
 @implementer(IPloneRelease)
 class PloneRelease(Item):
+    """ """
+    # @property
+    # def title(self):
+    #     import pdb; pdb.set_trace( )
```

Comment:
@gforcada maybe it's probably something we can live with it ;) However, @esteele could you amend it, so we can merge it right away. I can update plone.org too when it's merged.